src/app/destinyTrackerApi/d2-reviewDataCache.js

```diff
@@ -44,11 +44,29 @@ class D2ReviewDataCache {
     return rating.toFixed(1);
   }
 
+  _getDownvoteMultiplier(dtrRating) {
```

Comment:
love this idea. how could it scale? over time everything may have over 300 ratings. similarly if we reset scores per-season everything will have a -2.5 multiplier. 

could we build a scale based off of the average number of all reviews or something? 🤔

Reply:
i realize we're using the term scale in like 3 different ways here... 😬 sorry

Reply:
As we slice things more finely (platforms, game mode, sandbox rebalances), we'll be dealing with smaller sets of ratings, so I'm not expecting to have huge numbers of ratings for everything.  Rather than use these hard magic numbers, we could find the maximum number of votes that a given item has and base the weighting scale off of that.

I'm also OK with things looking curiously bad, because I think that will spur people to throw a thumbs up at it.

Webscale.

Reply:
OK, a little new logic is in there - rather than fixed points, it weights based off of total number of votes compared to the max in the user's data.